lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/domain/CredentialEvidenceItem.java

```diff
@@ -21,19 +21,21 @@ public class CredentialEvidenceItem {
     private List<DcmawCheckMethod> failedCheckDetails;
     private List<String> ci;
 
-    public CredentialEvidenceItem(EvidenceType evidenceType, int score) {
+    public CredentialEvidenceItem(EvidenceType evidenceType, int score, List<String> ci) {
         if (EvidenceType.ACTIVITY.equals(evidenceType)) {
             this.activityHistoryScore = score;
         } else if (EvidenceType.IDENTITY_FRAUD.equals(evidenceType)) {
             this.identityFraudScore = score;
         } else if (EvidenceType.VERIFICATION.equals(evidenceType)) {
             this.verificationScore = score;
         }
+        this.ci = ci;
```

Comment:
```suggestion
    public CredentialEvidenceItem(EvidenceType evidenceType, int score) {
        if (EvidenceType.ACTIVITY.equals(evidenceType)) {
            this.activityHistoryScore = score;
        } else if (EvidenceType.IDENTITY_FRAUD.equals(evidenceType)) {
            this.identityFraudScore = score;
        } else if (EvidenceType.VERIFICATION.equals(evidenceType)) {
            this.verificationScore = score;
        }
        this.ci = Collections.emptyList();
```
I think this constructor is only ever being used to pass in an empty list

Reply:
I kept this in more for potential future use in case we ever need to set this value for future CRI's. Plus I do use it at the moment during tests to setup test items with CI's